src/node-pool/index.ts

```diff
@@ -0,0 +1,164 @@
+/**
+ * NodePool module
+ * @module @aeternity/aepp-sdk/es/node-pool
+ * @export NodePool
+ * @example import { NodePool } from '@aeternity/aepp-sdk'
+ */
+import stampit from '@stamp/it'
+// @ts-expect-error TODO remove
+import { getNetworkId } from '../node'
+import { DisconnectedError, DuplicateNodeError, NodeNotFoundError } from '../utils/errors'
+
+// TODO: Update me when node module is migrated to TS
+type Node = any
+
+interface NodeInfo {
+  name: string
+  version: string
+  networkId: string
+  consensusProtocolVersion: string
+}
+
+interface NodePool {
```

Comment:
I didn't get why do we need a separate interface, can we use something like `InstanceType<typeof  _NodePool>` instead?